apps/pp/views/references.py

```diff
@@ -161,6 +169,6 @@ def get(self, request, *args, **kwargs):
 
         queryset = self.annotate_fetched_queryset(queryset)
 
-        data_list = self.preserialize_queryset(queryset)
+        data_list = self.pre_serialize_queryset(queryset)
```

Comment:
I like more the correct `preserialize_queryset` version :peanuts:

Reply:
what do you mean correct? :smile: any of my dictionaries don't know word preserialize :tongue: